fuel/transformers/__init__.py

```diff
@@ -619,3 +619,29 @@ def get_data(self, request=None):
             raise ValueError
         data = next(self.child_epoch_iterator)
         return data
+
+
+class SourcesSubsetTransformer(Transformer):
+    """Selects a subset of the stream sources.
+
+    Parameters
+    ----------
+    data_stream : :class:`DataStream` or :class:`Transformer`.
```

Comment:
I think this would also work with `ServerDataStream` out of the box. What do you think of :class:`AbstractDataStream` as an alternative type for that argument?